tests/features/test_secondary_structure.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+
+from deeprankcore.domain import nodestorage as Nfeat
+from deeprankcore.features.secondary_structure import (SecondarySctructure,
+                                                       _classify_secstructure,
+                                                       add_features)
+
+from . import build_testgraph
+
+
+def test_secondary_structure_residue():
+    test_case = '9api' # properly formatted pdb file
+    pdb_path = f"tests/data/pdb/{test_case}/{test_case}.pdb"
```

Comment:
Here we could use PATH_TEST global tests variable to refer to "tests/" folder (`os.join(PATH_TEST, 'data', 'pdb', f'{test_case}', f'{test_case}.pdb')`)